tools/msconvert/msconvert_macros.xml

```diff
@@ -39,10 +39,12 @@
       ln -s '$data_processing.precursor_refinement.input_ident' '$input_ident_name' &&
     #end if
 
-    uid=`id -u` &&
-    gid=`id -g` &&
+    ## create a writable copy of wine prefix
+    ## (since copying fails for some html files we swallow the exit code)
+    export WINEPREFIX=\$(mktemp -d) &&
+    (cp -ar /wineprefix64/* \$WINEPREFIX || true) &&
```

Comment:
I think this is creating a lot of messages on the stderr. Please look at the planemo outputs.

Reply:
OK somehow I missed this part. Hmm. So this is going to copy WINEPREFIX for every job even it runs on the same node. Is it feasible to cache it on the instance so multiple jobs run in series don't pay for the copy each time? That's how the mywine script works. I can definitely imagine race condition issues with that though.

Reply:
Might be an option. Question if is if actually something is written into `WINEPREFIX` or if the necessary write access is just a technical necessity. 

Also for Galaxy's singularity support I do not like the fact that each user downloads a separate copy of the container. But maybe this is avoidable via job_conf.xml (but probably only for a single container).. lets see.

Reply:
@chambm could you provide me a link explaining a bit more on the `mywine` script?

Reply:
btw. on my system cpopying needs ~14s

Reply:
> I think this is creating a lot of messages on the stderr. Please look at the planemo outputs.

So I redirect to `/dev/null` now. But maybe its better for future debugging to redirect to some file?